routes/web.php

```diff
@@ -18,5 +18,6 @@
     Route::get('/', 'HomeController@home')->name('home');
     Route::group(['prefix' => 'admin'], function() {
         Route::get('/', 'Admin\DashboardController@index')->name('admin.dashboard');
+        Route::resource('product', 'Admin\ProductController')->names('products');
```

Comment:
```suggestion
        Route::resource('products', 'Admin\ProductController');
```
tự có name rồi, em thêm làm gì nữa

Reply:
Em dùng route group thì phải định danh lại nó ạ. vì xem vẫn được nhưng khi em sửa sản phẩm thì sẽ báo route không tìm thấy (qicktask em bị dính và em bỏ route group ạ). 
đây là link em tìm thấy lỗi và em khắc phụ như vậy ạ. và các pull sau em sử dụng vẫn chạy tốt ạ.
https://github.com/laravel/framework/issues/3370